deploy/olm-catalog/windows-machine-config-operator/manifests/windows-exporter-role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
```

Comment:
now there's the final newline missing :D (sorry to bug y'all with this)

Reply:
whoops, adding those back in now.. Thanks for catching this!